py/desispec/pixgroup.py

```diff
@@ -181,9 +181,11 @@ def read(cls, filename):
         nspec = len(fibermap)
         night = np.tile(header['NIGHT'], nspec).astype('i4')
         expid = np.tile(header['EXPID'], nspec).astype('i4')
+        mjd = np.tile(header['MJD-OBS'], nspec).astype('f8')
```

Comment:
MJD-OBS is a pretty fundamental keyword that should be in every exposure, but are we confident enough of this card's existence that we want the code to fail if it isn't there?

Reply:
Good point.  I updated code to support "MJD-OBS" (current data), "MJD" (older sims), and not crash if both are missing (still propagates an MJD=-1 column to keep the data model consistent).